keylime-agent/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+authors = ["Keylime Authors"]
+edition = "2018"
```

Comment:
maybe is time to move it to 2021 like the keylime crate? or there is a dependency on the minimal version used for the compiler?

Reply:
That sounds like a good idea.